filecoin-proofs/src/bin/paramcache.rs

```diff
@@ -24,6 +24,7 @@ const PUBLISHED_SECTOR_SIZES: [u64; 10] = [
     SECTOR_SIZE_16_MIB,
     SECTOR_SIZE_512_MIB,
     SECTOR_SIZE_1_GIB,
+    SECTOR_SIZE_4_GIB,
```

Comment:
why are we publishing this size?

Reply:
This was just to help GPU testing 4GiB trees.  A bit quicker than 32GiB ;-)  I don't mind either way if it stays or goes though.